src/notebooklm/auth.py

```
@@ -199,52 +199,89 @@ def _validate_required_cookies(
+# YouTube / Docs / Mail / myaccount cookies do NOT appear in any traced
+# flow. They are now :data:`OPTIONAL_COOKIE_DOMAINS` — opted in via
+# ``notebooklm login --include-domains=...``. This narrows the blast
+# radius if ``storage_state.json`` is ever leaked (synthesis K15).
+#
+# REQUIRED is also fed verbatim to ``rookiepy.load(domains=...)`` by
+# ``_login_with_browser_cookies`` (via
+# :func:`notebooklm.cli.session._build_google_cookie_domains`); adding a
+# domain here automatically extends what we ask the browser for at login.
+# This is the single load-bearing T5.G control: blast-radius reduction
+# is enforced at extraction time (what rookiepy returns), not at the
+# runtime gate, which stays permissive over the REQUIRED ∪ OPTIONAL
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `∪` (UNION). Did you mean `U` (LATIN CAPITAL LETTER U)?

(RUF003)